live-examples/css-examples/basic-box-model/display.html

```diff
@@ -38,11 +38,13 @@
 <div id="output" class="output large hidden">
   <section id="default-example" class="default-example">
     <div class="example-container">
```

Comment:
> Yes, we need to be more explicit.

@OnkarRuikar, should we add this then:

```suggestion
<div id="output" class="output large hidden">
  <p>The <code>display</code> property is set on the <code>div</code> element enclosed by the orange dashed border.
    You can apply different <code>display</code> values on this element to observe their effects on its layout
    in relation to adjacent elements.
    Note that this element contains three child <code>div</code> elements.
  <section id="default-example" class="default-example">
    <div class="example-container">
```



This is how it would look:

<kbd><img width="1013" alt="display-inetractive" src="https://github.com/mdn/interactive-examples/assets/23019147/7b23c12f-60e2-45ca-8da2-58e00fa11eb6"></kbd>

Reply:
@OnkarRuikar I didn't realize you opened another PR to add the statement

Reply:
Oh, that PR is in `mdn/content`.
I have not noticed other instances where we discuss or refer to the "Try it" demo examples in content.
It might be best to keep the interactive demo examples self-contained. Let me know what you think - we can close https://github.com/mdn/content/pull/31672 and instead make the explanatory updates (as suggested above) to the code within this repo.

Reply:
@dipikabh that's too much text in the output. On narrow screens, readers will have to scroll to see the actual target div.

> I have not noticed other instances where we discuss or refer to the "Try it" demo examples in content.

We do have some "Try it" sections with pros, e.g. https://developer.mozilla.org/en-US/docs/Web/CSS/max#try_it

In the `mdn/content` PR we can put the text below the macro call. WDYT?

Reply:
I am still trying to see if we can maintain this info in the example itself. How about using the text from your content PR:
<kbd>![display-interactive-2](https://github.com/mdn/interactive-examples/assets/23019147/40d78ac4-06c8-444f-b9e8-dca907b6e1dd)</kbd>

or even a more shortened version:

<kbd>![display-interactive-4](https://github.com/mdn/interactive-examples/assets/23019147/eff184a4-da03-489d-b6c5-0b7f45b705be)</kbd>

Else the last option is to add text in content after the macro call.

Reply:
Let's pic the shortest version `Apply different <code>display</code> values on the dashed orange-bordered <code>div</code>, which contains three child elements.`